AlloyReact/Static/react/ReactNumber.jsx

```diff
@@ -1,44 +1,11 @@
 import React from "react";
 import Clock from "./Clock";
+import editable from './editable';
 
-export default class ReactBlock extends React.Component {
+const ReactBlock = ({ propertyName, number }) => 
+    <div>
+        The current value of the <strong>{propertyName}</strong> property is <strong>{number}</strong>, and the current time is: <strong><Clock /></strong>
```

Comment:
Minor thing: can you keep the first sentence as well? It's there to show the difference in timestamps between first render and future updates, to show how the component is updated more efficiently with the contentSaved event.

`This React component was rendered at <strong>{this.state.firstRender.toLocaleTimeString("sv-SE")}</strong> with a number of <strong>{this.props.someNumber}</strong>.<br/><br/>`